Assets/Tests/InputSystem/Unity.InputSystem.Tests.asmdef

```diff
@@ -40,6 +40,11 @@
             "name": "Unity",
             "expression": "2022.1",
             "define": "TEMP_DISABLE_STANDALONE_OSX_XINPUT_TEST"
+        },
+        {
+            "name": "Unity",
+            "expression": "2020.2",
```

Comment:
Do we want to enable this support all the way back to 2020.2? 
Is that valid? Does the UI TK feature set for the editor side work that far back?